### ISSUE TITLE
Remove links to talk pages from old schedule views

### ISSUE THREAD

**Issue by rixx** [pretalx/pretalx#281]
## Expected Behavior
I don't want to be linked to 404 pages.

## Current Behavior
If I browse an older schedule version and follow the URL to a now descheduled talk, it will 404. Instead I'd prefer to have no URLs at all in archived schedules. Additionally, deleted submissions should be displayed as "[deleted]".

**Comment by rahmonov** [comment]
Hey guys, can I be assigned to this?

**Comment by luto** [comment]
@rixx did you start working on this yet? if not, sure, go ahead @rahmonov! looking forward to the PR.

**Comment by rixx** [comment]
Hey @rahmonov, this issue is untaken, I'll change its state to "wip" to reflect that you're working on it. 👍

**Comment by rahmonov** [comment]
@rixx do "descheduled", "archived" and "deleted" all refer to "talk.state == 'deleted'"? I am just not sure if I understood it correctly. Deleted submissions should not have links and be displayed "[deleted]". Is this correct?

**Comment by rixx** [comment]
@rahmonov Aw damn, I was too vague in my phrasing. I was referring to the *public* schedule (that resides in the `agenda/` app).

- With "descheduled talk" I referred to talks that once were part of a schedule, but are not part of the latest schedule – those talks will not have a talk page, but the agenda view will contain a link to a talk page.
- With "archived schedule" I referred to a schedule version that is neither the current release, nor the WIP version, but an older version.

I'm sorry for not making these things clear right away – spending so much time with the code is not great for ones communication skills, it seems. ;)
